source/_patterns/components/04-toast-notifications/00-toast-notification.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="pf-toast alert {{ alertClass }} {{#if alertDismissible}}alert-dismissible{{/if}}" role="alert">
+{{#if alertDismissible}}
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
The X icon is something we will reuse on other patterns, we should use a button icon